setup.py

```diff
@@ -65,5 +65,6 @@
                       "pytest-dependency", "tomli"],
               "hard-encoding-detection": ["chardet"],
               "toml": ["tomli"],
+              "shtab": ["completion"],
```

Comment:
I'm pretty certain that's the wrong way round and it should be:
friendly name: list of packages that are required
```suggestion
              "completion": ["shtab"],
```

Reply:
Fixed. It looks like I am so tired when I write this line.